core/system/rule.go

```diff
@@ -71,18 +74,20 @@ func (t AdaptiveStrategy) String() string {
 }
 
 type Rule struct {
-	ID           string           `json:"id,omitempty"`
-	MetricType   MetricType       `json:"metricType"`
-	TriggerCount float64          `json:"triggerCount"`
-	Strategy     AdaptiveStrategy `json:"strategy"`
+	ID            string           `json:"id,omitempty"`
+	MetricType    MetricType       `json:"metricType"`
+	TriggerCount  float64          `json:"triggerCount,omitempty"`
+	Strategy      AdaptiveStrategy `json:"strategy"`
+	LowWaterMark  int64            `json:"lowWaterMark,omitempty"`
```

Comment:
Maybe we need to keep naming convention about triggered threshold.
such as: "LowThreshold" and "HighThreshold".